Uppsala/UppEt20.xml

```diff
@@ -0,0 +1,524 @@
+                                    ንሴብሐከ፡ እግዚኦ፡ ለአኀዜ፡ ኵሉ፡ አቡሁ፡ ለእግንዚእነ፡ ወአምላክነ፡ 
+                                    ወመድኃኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ንስእለከ፡ ሕገ፡ ዘሥጋ፡ አናሕሲ፡ ለነ፡ 
+                                    ወለዓለምሰ፡ ኢትትመዓዓነ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                    ወአብዝኅ፡ ምሕረትከ፡ ላዕሌነ፡ ወፍጡነ፡ ፈኑ፡ ሣህለከ፡ ዲቤነ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                </incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.8.6">
+                                <locus from="185va"/>
+                                <title type="complete"></title>
+                                <incipit>
+                                    <locus from="185va" to="185vb"/>
+                                    <hi rend="rubric">ወእምዝ፡</hi> ጸለየ፡ <sic corresp="JK">ኢ</sic>ርምያስ፡
+                                    በቤተ፡ መቅደስ፡ እንዘ፡ ይብል፡ ቅዱስ፡ ቅዱስ፡ ቅዱስ፡ አንተ፡ መዓዛ፡ ጥዑ<pb n="185vb"/>ም፡ 
+                                    ለሰብእ፡ ወብርሃን፡ ዘበአማን፡ ዘያበርህ፡ ሊተ፡ እስከ፡ አነ፡ እበጽሕ፡ 
+                                    ቅድሜከ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                </incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.8.7">
+                                <locus from="186r"/>
+                                <title type="complete">Prayer of balm and incense</title>
+                                <incipit>
+                                    <hi rend="rubric">ጸሎት፡</hi> በእንተ፡ በለሳን፡ ወዕጣን<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                    ነአኵተከ፡ አቡሁ፡ ለእግዚእነ፡ ወመድኃኒነ፡ ኢየሱስ፡ ክርስቶስ፡ በእንተ፡ ቅድስት፡ ጼና፡ አኮ፡ ባሕቲታ፡ አላ፡ 
+                                    ዘፍጥረት፡ አላ፡ ዘመንፈስ፡ ለነ፡ ዘፄና፡ ወሀብከነ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                </incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.8.8">
+                                <locus from="186rb"/>
+                                <title type="complete"></title>
+                                <incipit>
+                                    <hi rend="rubric">ጸሎት፡</hi> ለተመጥዎ፡ ስእለት፡ እግዚአብሔር፡ እግዚእየ፡ ኢየሱስ፡ ክርስ፡ መንፈስ፡ 
+                                    ቅዱስ<hi rend="rubric" rendition="#partialRubric">፤</hi> ስማዕ፡ እግዚኦ፡ ዘንተ፡ ስእለትየ፡ 
+                                    ወፈጽሞ፡ ሊተ፡ ጸሎትየ፡ ወፈቃድየ፡ ለገብርከ፡ ብዕለ፡ ስብሐቲሁ፡ በብዝኃ፡ ምሕረትከ፡ ወዘኒ፡ 
+                                    ዘወሀብከኒ፡ ኢትሂደኒ<hi rend="rubric" rendition="#partialRubric">፤</hi>
+                                </incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.8.9">
+                                <locus from="186va"/>
+                                <title type="complete">Prayer for the deceased</title>
```

Comment:
different from LIT6452Prayer?

Reply:
I don't know how to decide this. The incipit of LIT6452Prayer in BLorient546 is identical to the incipit of LIT2252salota#Dead, so they may well be duplicates

Reply:
In this case we simply need to add `corresp="LIT6452Prayer"` to the div with xml:id="Dead" in the record `LIT2252salota`, showing that this part of LIT2252salota can circulate as LIT6452Prayer. Whenever LIT6452Prayer occurs outside of LIT2252salota one uses 6452, when it is part of 2252 one uses LIT2252salota#Dead

Reply:
That sounds good! I will add it when I add the incipits